lbry/extras/cli.py

```diff
@@ -294,6 +297,10 @@ def main(argv=None):
     for directory in (conf.data_dir, conf.download_dir, conf.wallet_dir):
         ensure_directory_exists(directory)
 
+    total, used, free = shutil.disk_usage('/')
```

Comment:
free disk space should not be required to start the sdk. and even if we did want to require this, your check misses some edge cases: maybe the download directory is on a different disk than `/`, or maybe the path is symlinked to elsewhere.

if you want to cover these cases, then it might be a worthwhile add. otherwise i'd suggest removing this check altogether.

Reply:
Yeah, after I made the PR I thought about the download directory being on a different disk, etc. I only included this because of the comment on the original issue "bonus: also check if disk is full when sdk starts. this check should be outside ensure_directory_exists(). it probably goes somewhere near here". I'll update the PR and remove this.

Reply:
Changed my mind, I decided to try to do the checks on the directories properly. I used an IOError for the failure, which I'm not 100% sure if that's the wrote one to throw, but it seemed reasonable. Let me know if this needs any other changes!